raiden/network/rpc/client.py

```diff
@@ -283,7 +283,7 @@ def get_transaction_receipt(self, tx_hash: bytes):
         return self.web3.eth.getTransactionReceipt(encode_hex(tx_hash))
 
     def get_transaction_from(self, tx_hash: bytes) -> Optional[Address]:
-        receipt = self.web3.eth.getTransactionReceipt(encode_hex(tx_hash))
+        receipt = self.web3.eth.getTransaction(encode_hex(tx_hash))
```

Comment:
why this change?

Reply:
tester `getTransactionReceipt` doesn't have `from` key. transaction does.